src/support/stat.c

```diff
@@ -685,10 +685,13 @@ static const char *const __stats_connection_desc[] = {
   "cache: forced eviction - pages selected unable to be evicted time",
   "cache: hazard pointer blocked page eviction", "cache: hazard pointer check calls",
   "cache: hazard pointer check entries walked", "cache: hazard pointer maximum array length",
+  "cache: history store key truncation due to mixed timestamps",
+  "cache: history store key truncation due to the key being removed from the page",
```

Comment:
How about mentioning the data page here? 
`cache: history store key remove due to the key being removed from the data page`

Reply:
554ee31